File: Anul II/Medii de proiectare si programare/Laborator/lab07/Ubb.BikeContest.Networking/Request/GetRacesByEngineCapacityRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ubb.BikeContest.Networking.Request
{
    [Serializable]
    public class GetRacesByEngineCapacityRequest : IRequest
    {
        private readonly int engineCapacity;

        public GetRacesByEngineCapacityRequest(int engineCapacity)
        {
            this.engineCapacity = engineCapacity;
        }

        public virtual int EngineCapacity { get {  return engineCapacity; } }
    }
}

File: Anul II/Medii de proiectare si programare/Laborator/lab09/networking/src/main/java/com/ubb/protocol/request/GetRacesByEngineCapacityRequest.java
package com.ubb.protocol.request;

public class GetRacesByEngineCapacityRequest implements Request {
    private final Integer engineCapacity;

    public GetRacesByEngineCapacityRequest(Integer engineCapacity) {
        this.engineCapacity = engineCapacity;
    }

    public Integer getEngineCapacity() {
        return engineCapacity;
    }
}
